### PR TITLE
Wechall/additional

### PR DESCRIPTION
**Pull request by Zeeshan12340**
basically, adding the extra stuff for wechall.net #184 , global rank and user count, `global_scoreboard_data` is the same as `dojo_scoreboard_data` with arguments ignored and duration filter query removed so maybe it's not even needed. Other things worked fine for me and i tested for some edge cases but i obviously don't have the actual pwn.college dataset.

P.S: I don't know why these previous commits are still here.

### CODE REVIEW

**Comment by ConnorNelson** [comment]
So `ignore_visibility` actually has to do with visibility start/stop times. What we really want to do if we want official only, is `.filter_by(official=1)`. I'm also not completely sure what's going to happen when two official dojos share a challenge, but *probably* it will result in duplicated solves in this query. I think we need some sort of `.distinct(Solves.challenge_id)` or something like that before we group.

**Comment by Zeeshan12340** [comment]
Whoops, completely misread what `ignore_visibility` was doing. `DojoChallenges.solves().filter(Dojos.official == 1).distinct(Solves.challenge_id)`  seems like it filters for the official dojos correctly. I'll have to test the `distinct ` query because I do not have dojos that share challenges setup.

File: dojo_plugin/api/v1/score.py [hunk 1]
```diff
@@ -44,4 +59,9 @@ def get(self):
         user_score = sum(1 for solve in user.solves if solve.challenge_id in official_challenge_ids)
         max_score = len(official_challenge_ids)
 
-        return f"{user_score}:{max_score}"
+        user_count = len(Users.query.all())
```
Comment (ConnorNelson) [inline dojo_plugin/api/v1/score.py:62]: Can we do `Users.query.count()`

```
In [77]: %time Users.query.count()
CPU times: user 4.96 ms, sys: 66 µs, total: 5.02 ms
Wall time: 18.7 ms
Out[77]: 38869

In [78]: %time len(Users.query.all())
CPU times: user 5.13 s, sys: 281 ms, total: 5.41 s
Wall time: 5.42 s
Out[78]: 38869
```

**Review by ConnorNelson — CHANGES_REQUESTED** [review]
Happy to merge with this simple change. Thanks!

File: dojo_plugin/api/v1/score.py (hunk 1 above)
Comment (Zeeshan12340) [reply dojo_plugin/api/v1/score.py]: Done!

**Pull request merged**